runway/blueprints/staticsite/dependencies.py

```diff
@@ -110,17 +115,30 @@ def create_template(self):
             Value=artifacts.ref()
         ))
 
+        callbacks = self.context.hook_data['aae_callback_url_retriever']['callback_urls']
+
         if variables['AuthAtEdge']:
+            user_pool_id = variables['UserPoolId']
+
+            if variables['CreateUserPool']:
+                user_pool = template.add_resource(
+                    cognito.UserPool("AuthAtEdgeUserPool")
+                )
+
+                user_pool_id = user_pool.ref()
+
+                template.add_output(Output(
+                    'AuthAtEdgeUserPool',
```

Comment:
Also just style/preference -- let's call this `UserPoolId`

i.e. appending the value type, so its clear/unique compared to others we might add in the future like `UserPoolArn`. Nothing would be wrong about just having `UserPool` & `UserPoolArn`, but my preference is just to start explicit

Reply:
In regards to above should we make this `AuthAtEdgeUserPoolId`?